.github/workflows/build-and-deploy.yml

```
@@ -9,7 +9,7 @@ on:
     branches:
       - main
   release: # Trigger on release creation
-    types: [created]
+    types: [ created ]
```

yamllint

[error] too many spaces inside brackets

(brackets)